signac/contrib/job.py

```diff
@@ -606,6 +606,15 @@ def data(self, new_data):
         """
         self.stores[self.KEY_DATA] = new_data
 
+    @property
+    def project(self):
+        """Return the project that contains this job."""
+        return self._project
+
+    @project.setter
+    def project(self, new_project):
+        raise ValueError("Setting project attribute directly is not allowed")
+
```

Comment:
Just delete this -- without a `setter`, the property will not be settable and will raise an `AttributeError`.

```suggestion
```